app/models/manageiq/providers/oracle_cloud/inventory/persister/definitions/cloud_collections.rb

```diff
@@ -2,8 +2,16 @@ module ManageIQ::Providers::OracleCloud::Inventory::Persister::Definitions::Clou
   extend ActiveSupport::Concern
 
   def initialize_cloud_inventory_collections
-    %i(vms).each do |name|
-      add_collection(cloud, name)
+    add_collection(cloud, :cloud_tenants) do |builder|
+      builder.add_default_values(:ems_id => ->(persister) { persister.manager.id} )
     end
+    add_collection(cloud, :flavors)
+    add_collection(cloud, :hardwares)
+    add_collection(cloud, :miq_templates) do |builder|
+      builder.add_properties(:model_class => ::ManageIQ::Providers::OracleCloud::CloudManager::Template)
```

Comment:
This is a pet peeve of mine, the auto_model_class doesn't work because the association is called `miq_templates` and the class name isn't `...OracleCloud::CloudManager::MiqTemplate` but every cloud provider follows this pattern so we should handle this in core

Reply:
makes sense